mmv1/products/dlp/DiscoveryConfig.yaml

```diff
@@ -346,6 +389,17 @@ properties:
                         - :UPDATE_FREQUENCY_NEVER
                         - :UPDATE_FREQUENCY_DAILY
                         - :UPDATE_FREQUENCY_MONTHLY
+                - !ruby/object:Api::Type::NestedObject
+                  name: inspectTemplateModifiedCadence
+                  description: Governs when to update data profiles when the inspection rules defined by the `InspectTemplate` change. If not set, changing the template will not cause a data profile to update.
+                  properties:
+                    - !ruby/object:Api::Type::Enum
+                      name: 'frequency'
```

Comment:
This should probably be marked as required

Reply:
Why should this be required? It's not required in the API.

Reply:
We'll handle an empty value in the API as if it was set to never, but we won't actually return a server-side value.

Reply:
Generally we prefer to prevent users from specifying empty objects in Terraform. It can have some unexpected behaviors, so marking at least one sub-field as required prevents that.

If there's not a valid reason for a user to specify 
```
inspect_template_modified_cadence {}
```

Then it's easier to just mark the sub-field as required. It only becomes required in their config *if* they specify the inspect_template_modified_cadence field in the first place

Reply:
Got it. Added 'required: true' to the frequency value.